packages/core/test/NumberField_test.js

```diff
@@ -274,7 +278,7 @@ describe("NumberField", () => {
             sinon.assert.calledWithMatch(onChange.lastCall, {
               formData: test.output,
             });
-            expect($input.value).eql(test.input);
+            expect($input.getAttribute("value")).eql(test.input);
```

Comment:
Not entirely sure why the behaviour of `input.value` has changed, but using `input.getAttribute("value")` and `input.defaultValue` still seems to work as expected. Chose to use `input.getAttribute("value")` instead of digging deeper of what's up with the input.value regression.